DNSExfiltration.md

```diff
@@ -14,6 +14,9 @@ This is a common theme where an attacker gets specific information about where t
 
   <img src="https://raw.githubusercontent.com/step-security/supply-chain-goat/main/images/DNSExfiltration.png" alt="DNS exfiltration" width="800">
 
+## How StepSecurity mitigates threat?
```

Comment:
Please reword it to `How does StepSecurity mitigate this threat?`

Reply:
@varunsh-coder made all the changes, we're good to go